### DISCUSSION TITLE
Symbols from TCP Data

### DISCUSSION THREAD

**Discussion by kjacksonmusic** [onlxltd/CommsKit#6] [Bug Reports]
Hello!

I am using commskit to intercept TCP messages from a device controller and it seems as though Commskit is not able to interpret the symbols properly? 

Big Sur 11.5
Commskit 1.5.0 build 72c3bc0
<img width="698" alt="Screen Shot 2021-08-12 at 7 47 53 AM" src="https://user-images.githubusercontent.com/2052863/129192043-0eee9555-c163-4ef7-85e8-d1b0640f8e0a.png">

**Comment by mdidon** [comment]
@kjacksonmusic  CommsKit decodes the data to a string using utf8 so I would assume it's either using a different encoding type or is raw data.

I have created a beta build for you which includes a function to set the encoding then let us know your findings.

This link expires in 1 hour and the macOS build is signed but is not notarised. https://url.dev/CN2ZXrh/

<img width="694" src="https://user-images.githubusercontent.com/16917736/129215213-dae486d7-79d2-445e-a454-9cdc5db9e130.png">

**Reply by kjacksonmusic** [reply]
This worked, thank you, 
<img width="540" alt="worked" src="https://user-images.githubusercontent.com/2052863/129357634-48a86514-4f36-4af4-bd5e-9becff32b1ca.png">

**Reply by mdidon** [reply]
@kjacksonmusic Perfect, we'll include this function with the next release. Thanks for the update

**Comment by kjacksonmusic** [comment]
Hey Matt can you resend the link.

**Reply by mdidon** [reply]
@kjacksonmusic https://url.dev/CGf3x2b/

**Comment by kjacksonmusic** [comment]
Got it, thanks!
Kevin Jackson
